.circleci/config.yml

```diff
@@ -67,21 +67,27 @@ commands:
       - run:
           name: 'Check Config'
           command: ./.circleci/check_config.sh
-      - run:
-          name: 'Setup Google Cloud Storage'
-          command: ./.circleci/setup_storage.sh
       - run:
           name: 'Configure Hosts'
           command: cat ./build-system/test-configs/hosts | sudo tee -a /etc/hosts
       - run:
           name: 'Install Dependencies'
           command: ./.circleci/install_dependencies.sh
+      - run:
+          name: 'Restore Build Output'
+          command: ./.circleci/restore_build_output.sh
   teardown_vm:
     steps:
       - persist_to_workspace:
           root: /tmp
           paths:
             - workspace
+  setup_gcloud_storage:
```

Comment:
Shouldn't we be able to remove all gcloud references since we're using circleci storage?

Reply:
Yes, I think I prefer to make that final switch in a later PR due to the complexity of the migration :)